game/views.py

```
@@ -403,222 +447,262 @@ def ai_move(request):
+    winner = "black" if resigning_player == "white" else "white"
+
+    game_status = "resignation"
 
     game.game_status = game_status
-    request.session['game'] = game.to_dict()
+    request.session["game"] = game.to_dict()
     request.session.modified = True
 
     try:
         record_game_result(request, game.mode, winner, 'resign', game.player_color)
     except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)